neurodamus/connection_manager.py

```diff
@@ -533,9 +535,10 @@ def connect_all(self, weight_factor=1, only_gids=None):
             weight_factor: Factor to scale all netcon weights (default: 1)
             only_gids: Create connections only for these tgids (default: Off)
         """
-        if self._synapse_counter is not None:
+        if SimConfig.dry_run:
             counts = self._get_conn_stats(self._src_target_filter, None)
-            self._synapse_counter.update(counts)
+            print("Stats:", self._dry_run_stats)
```

Comment:
This seems to just print random memory location for the DryRunStats object.
```
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f27f70>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f27f70>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f2bfa0>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f2bfa0>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f9c040>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f9c040>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f9c040>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f9c040>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f9c040>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f9c040>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f9c040>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f9c040>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f27f70>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f27f70>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f27fa0>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f27fa0>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f9c040>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f9c040>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f9c040>
Stats: <neurodamus.utils.memory.DryRunStats object at 0x2b0972f9c040>
```
Maybe it can be removed?

Reply:
Yeah, I was missing some final cleanup. Should be good now